### PR TITLE
TINY-9932: Add itemprop, itemscope, and itemtype to HTML5 schema

### PR DESCRIPTION
**Pull request by ArvinJ-H**
Related Ticket: 
TINY-9932

Description of Changes:
- Add itemprop, itemscope, and itemtype to HTML5 schema
- Update test

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

**Review by TheSpyder — APPROVED** [review]

File: .changes/unreleased/tinymce-TINY-9932-2024-03-06.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: Included itemprop, itemscope, itemtype as valid HTML5 element
```
Comment (spocke) [inline .changes/unreleased/tinymce-TINY-9932-2024-03-06.yaml:3]:
```suggestion
body: Included itemprop, itemscope, itemtype as valid HTML5 attributes.
```

**Review by spocke — APPROVED** [review]

**Comment by TheSpyder** [comment]
- [ ] do not merge until after the 7.0 release

File: modules/dragster/src/test/ts/browser/datatransfer/DataTransferTest.ts [hunk 2]
```diff
@@ -222,8 +223,9 @@ describe('browser.dragster.datatransfer.DataTransferTest', () => {
       assert.strictEqual(transfer.files.length, 2, 'Should have same number of files');
 
       transfer.clearData();
-      if (isFirefox || isSafari) {
+      if (isFirefox || isSafari || isChrome) {
```
Comment (ltrouton) [inline modules/dragster/src/test/ts/browser/datatransfer/DataTransferTest.ts:226]: This is all browsers that we test now so I don't think we need the `if` check anymore?

File: .changes/unreleased/tinymce-TINY-9932-2024-03-06.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: Included itemprop, itemscope, itemtype as valid HTML5 attributes.
```
Comment (ltrouton) [inline .changes/unreleased/tinymce-TINY-9932-2024-03-06.yaml:3]:
```suggestion
body: Included itemprop, itemscope, itemtype as valid HTML5 attributes in the core schema.
```

File: modules/tinymce/src/core/test/ts/webdriver/keyboard/SpaceKeyTest.ts [hunk 4]
```diff
@@ -39,7 +43,7 @@ describe('webdriver.tinymce.core.keyboard.SpaceKeyTest', () => {
       if (isSafari) { // Split due to normalization issue. See TINY-8833
         TinyAssertions.assertContent(editor, '<p>s &nbsp;<span style="display: block;" contenteditable="false">a</span></p>');
       } else if (isFirefox) { // Split due to normalization issue. See TINY-8833
-        TinyAssertions.assertContent(editor, '<p>s&nbsp; <span style="display: block;" contenteditable="false">a</span></p>');
+        TinyAssertions.assertContent(editor, '<p>s&nbsp; <br><span style="display: block;" contenteditable="false">a</span></p>');
```
Comment (ltrouton) [inline modules/tinymce/src/core/test/ts/webdriver/keyboard/SpaceKeyTest.ts:48]: This seems odd a `br` was added. What change caused us to have to update these test cases?

Comment (tiny-ben-tran) [reply modules/tinymce/src/core/test/ts/webdriver/keyboard/SpaceKeyTest.ts:48]: @ArvinJ-H , if I remember correctly from the conversation with you, this only occurs on the most recent firefox. maybe adding a comment to explain why `br` is added here

**Review by tiny-ben-tran — COMMENTED** [review]
Are there tests that cover the attributes?

File: modules/tinymce/src/core/test/ts/webdriver/keyboard/SpaceKeyTest.ts (hunk 4 above)
Comment (ArvinJ-H) [reply modules/tinymce/src/core/test/ts/webdriver/keyboard/SpaceKeyTest.ts:48]: this test is skipped... waiting for these two ticket to be resolved
https://ephocks.atlassian.net/browse/TINY-10742
https://ephocks.atlassian.net/browse/TINY-10743

File: modules/dragster/src/test/ts/browser/datatransfer/DataTransferTest.ts (hunk 2 above)
Comment (ArvinJ-H) [reply modules/dragster/src/test/ts/browser/datatransfer/DataTransferTest.ts:226]: different browser still have different behaviours, mainly recorded in those two tickets
https://ephocks.atlassian.net/browse/TINY-10743
https://ephocks.atlassian.net/browse/TINY-10742

**Pull request merged**